delta/plugins/composite-views/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/compositeviews/indexing/CompositeIndexingStream.scala

```diff
@@ -100,6 +116,37 @@ final class CompositeIndexingStream(
         }
         streams.foldLeft[Stream[Task, Unit]](Stream.empty)(_ merge _)
       }
+    view.value.rebuildStrategy match {
```

Comment:
I think it would be useful to add an overall documentation to how composite views are handled (incl restart).